### PR TITLE
new package: wireless-tools

### PR DESCRIPTION
**Pull request by xeffyr**
Needed for #1665

### CODE REVIEW

File: packages/wireless-tools/Makefile.patch [hunk 1]
```diff
@@ -0,0 +1,25 @@
+diff -uNr wireless_tools.30/Makefile wireless_tools.30.mod/Makefile
+--- wireless_tools.30/Makefile	2009-11-24 20:53:40.000000000 +0200
++++ wireless_tools.30.mod/Makefile	2017-10-17 11:19:33.190969453 +0300
+@@ -5,11 +5,11 @@
+ ## Installation directory. By default, go in /usr/local.
+ ## Distributions should probably use /, but they probably know better...
+ ifndef PREFIX
+-  PREFIX = /usr/local
```
Comment (Auxilus) [inline packages/wireless-tools/Makefile.patch]: You can fix the prefix and cc in `termux_step_post_configure () ` by sed btw,

Comment (xeffyr) [reply packages/wireless-tools/Makefile.patch]: Prefix is set when passing **PREFIX="${TERMUX_PREFIX}"** to make in ```termux_step_make()```. Package builds correctly and everything is placed in ${PREFIX}.

Comment (Auxilus) [reply packages/wireless-tools/Makefile.patch]: @xeffyr have you tried `Wireshark ` on termux?

Comment (xeffyr) [reply packages/wireless-tools/Makefile.patch]: No.

**Comment by Auxilus** [comment]
@xeffyr I compiled the package just by `make && make install` I didn't event had to change the prefix in Makefile, all I did after make is `mv $PREFIX/sbin/* $PREFIX/bin` so making all these package and overriding the function seems pointless to me ( I may be wrong )

**Comment by Auxilus** [comment]
We only need to change the man dir, which we can do by `sed `

**Comment by xeffyr** [comment]
What is wrong with man dir ? In docker image everything builds fine.
```
builder@6011bc622914:~/termux-packages/debs$ dpkg-deb -R wireless-tools_30pre9_aarch64.deb p
builder@6011bc622914:~/termux-packages/debs$ cd p
builder@6011bc622914:~/termux-packages/debs/p$ ls
DEBIAN  data
builder@6011bc622914:~/termux-packages/debs/p$ cd data/data/com.termux/files/usr
builder@6011bc622914:~/termux-packages/debs/p/data/data/com.termux/files/usr$ ls
bin  share
builder@6011bc622914:~/termux-packages/debs/p/data/data/com.termux/files/usr$ ls share
man
builder@6011bc622914:~/termux-packages/debs/p/data/data/com.termux/files/usr$ ls share/man
man5  man7  man8
builder@6011bc622914:~/termux-packages/debs/p/data/data/com.termux/files/usr$ ls bin
ifrename  iwconfig  iwevent  iwgetid  iwlist  iwpriv  iwspy
builder@6011bc622914:~/termux-packages/debs/p/data/data/com.termux/files/usr$
```

**Comment by Auxilus** [comment]
I mean we can sed the man dir path rather including it in separate patch

**Comment by xeffyr** [comment]
Closing in favor of https://github.com/termux/termux-root-packages/pull/4

File: packages/wireless-tools/Makefile.patch [hunk 2]
```diff
@@ -0,0 +1,38 @@
+diff -uNr wireless_tools.30/Makefile wireless_tools.30.mod2/Makefile
+--- wireless_tools.30/Makefile	2009-11-24 20:53:40.000000000 +0200
++++ wireless_tools.30.mod2/Makefile	2017-10-17 11:55:09.020967925 +0300
+@@ -5,11 +5,11 @@
+ ## Installation directory. By default, go in /usr/local.
```
Comment (hugolep44) [inline packages/wireless-tools/Makefile.patch]: test

**Pull request closed without merging**